src/Messages.js

```diff
@@ -107,6 +107,41 @@ export default defineMessages({
         description: 'description',
         defaultMessage: 'Description'
     },
+    threatType: {
+        id: 'threatType',
+        description: 'Malware Threat Type',
+        defaultMessage: 'Threat Type'
```

Comment:
I am not sure if UX how designed, but there is a convention that label should not have all capital words. Only the first word should be capital and the rest needs to be small letters until a dot. Can we change **Threat Type** to **Threat type**

Reply:
Done